code/controllers/subsystem/atoms.dm

```diff
@@ -51,14 +51,15 @@ SUBSYSTEM_DEF(atoms)
 	CreateAtoms(atoms, atoms_to_return)
 	clear_tracked_initalize()
 
+	var/len = late_loaders.len
```

Comment:
bad var, also change the len check to length()

Reply:
The var is used to keep the test log for atoms late loaded working. Because atoms are removed from the late_loaders list as they are being removed, and all atoms in the list will be late loaded, if I don't store the initial length before the testing() call, it would always report 0 atoms late loaded, which is both inaccurate and gives no information.

Reply:
It's an unused var. It generates a warning. Also, it shouldn't be named `len`.

Reply:
It is used though? just a few lines down on line 62 is
`testing("Late initialized [len] atoms")`
Although I guess I should rename it, maybe there's a name collision or something

Reply:
This has been fixed